### PR TITLE
1907-06-17

### PR DESCRIPTION
**Pull request by laneydegroat**
Deliverable 3

### CODE REVIEW

File: 1907-06-17.xml [hunk 1]
```diff
@@ -0,0 +1,6060 @@
...
+                            <cell>Aug. 10th</cell>
+                            <cell>Aug. 14th</cell>
+                            <cell>Aug. 26th</cell>
+                        </row>
+                    </table>
+                    <p>Monthly connections from Port Said to the ports of the Red Sea and the
+                        Persian Gulf. </p>
+                    <p>Port Said to Colombo, China, and Japan; ay 4th, S.S. Silesia; June 3rd; S.S.
+                        Scandia; July 4th; S.S. Hamburg; August 3rd, S.S. Rhenania. Also occasional
+                        sailing from Port Said to Marseilles, Oran and London. Roomy cabins
```
Comment (huoyanzhizhou) [inline 1907-06-17.xml]: Here you start and end the page 5 division. Please check in the outline window and make sure your content are inside page divs.

Comment (huoyanzhizhou) [reply 1907-06-17.xml]: My comment is not shown in the correct line. See line 4568 in the file you uploaded.

Comment (huoyanzhizhou) [reply 1907-06-17.xml]: Also, your page 3&4 are inside the page 2 division. Please check them.

**Comment by whanley** [comment]
Fine work! A bit to do yet with your divs and other details (see below). Also, make sure that all of your OCR is corrected. Keep it up Laney!

- line 1895: this date should go in the previous div. The headline for this div is THE KHEDIVE AND HIS CRITICS
- line 1988: the slash just before the end of the tag `<div type="page" n="3"/>` closes it before it begins. Remove the slash and you will have a `</div>` close tag, which you can then move to the end of the page.
- 1989: the local and general section contains stories, each of them a `<div type="item">` with its own headline.
- 2079, 2161 close the div

- 2660 there's a template for cheap prepaid ads
- 2727 update to reflect your issue

- 4570 add element="wire" attribute to div, divide off the rest of the items within it.

- 5793 close the div

**Comment by whanley** [comment]
looks like you submitted the changes in https://github.com/dig-eg-gaz/content/pull/655, so I'm closing this pull request

**Pull request closed without merging**